### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by rae-stanton**
Working on controllers for recipe and ingredients - WIP

### CODE REVIEW

**Comment by rae-stanton** [comment]
@spaulsandhu

File: app/controllers/recipe_controller.rb [hunk 1]
```diff
@@ -1,2 +1,37 @@
 class RecipeController < ApplicationController
+  def new
+    @recipe = Recipe.new
+    @ingredients = Ingredient.all
```
Comment (spaulsandhu) [inline app/controllers/recipe_controller.rb]: Why are we setting the instance variable here for `all` Ingredients?

Would something like `@ingredients = @recipe.ingredients` make more sense so the ingredients we're instantiating for the view are scoped to the specific recipe we're viewing? 🤔

File: app/controllers/recipe_controller.rb [hunk 2]
```diff
@@ -1,2 +1,37 @@
...
+  end
+
+  def edit
+    @recipe = Recipe.find(params[:id])
+    @ingredients = Ingredient.all
```
Comment (spaulsandhu) [inline app/controllers/recipe_controller.rb]: Same as above, do we want to set the instance variable to `ALL` Ingredients in the db or just the ingredients associated with the recipe?

File: test/controllers/ingredient_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class IngredientControllerTest < ActionDispatch::IntegrationTest
```
Comment (spaulsandhu) [inline test/controllers/ingredient_controller_test.rb:3]: Do we plan on using minitest for this app or Rspec?

File: test/controllers/recipe_controller_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class RecipeControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
+end
```
Comment (spaulsandhu) [inline test/controllers/recipe_controller_test.rb:7]: Rails generates spec files for minitest when using generators, so we may want to re-assess adding these now if we plan on using `Rspec` for our testing purposes.

**Review by spaulsandhu — COMMENTED** [review]
This looks like a good start, but I think we have some clean-up and reorganizing of some of our logic to take care of before we can merge down. Lemme know if you want to chat about things or if my comments aren't clear and I'll be happy to go over details with you. 👍🏽

File: test/controllers/ingredient_controller_test.rb (hunk 3 above)
Comment (rae-stanton) [reply test/controllers/ingredient_controller_test.rb:3]: Going to add rspec for test suite~

File: app/controllers/recipe_controller.rb (hunk 1 above)
Comment (rae-stanton) [reply app/controllers/recipe_controller.rb]: ^ yeah that makes way more sense

**Comment by rae-stanton** [comment]
Leaving this open until rspec has been added versus minitest generated tests

File: test/controllers/recipe_controller_test.rb (hunk 4 above)
Comment (spaulsandhu) [reply test/controllers/recipe_controller_test.rb:7]: We should delete these files in a different PR.

**Review by spaulsandhu — APPROVED** [review]
LGTM

**Pull request closed without merging**